src/components/Post/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.scss';
+
+const Post = ({
+  title,
+  body,
+  publishedAt,
+  author: { name },
+}) => (
+  <div className="post">
+    <div className="post__header">
+      <div className="post__header__user">
+        <div className="post__header__username">
+          <div className="nickname">{`@${name}`}</div>
+        </div>
+      </div>
+    </div>
+    <div className="post__content">
+      <div className="post__content__title">
+        {title}
+      </div>
+      <div className="post__content__body">
+        {body}
+      </div>
+    </div>
+    <div className="post__footer">
+      <div className="post__footer__date">
+        <div className="date">{publishedAt.split('T')[0]}</div>
+      </div>
+    </div>
```

Comment:
Same here with the nestings. Do we really need 3 divs to show a date?

Also another thing without taking consideration the above comment, is that it is usually a good practice for readability to separate different parts of the render with an empty line for readability. For example the suggestion
```suggestion
    
    <div className="post__footer">
      <div className="post__footer__date">
        <div className="date">{publishedAt.split('T')[0]}</div>
      </div>
    </div>
```